InventoryComponents/InventoryHUD/InventoryHUDButtons.gd

```diff
@@ -0,0 +1,41 @@
+extends Control
```

Comment:
I would add a DEBUG_VAR and only allow this to be added to a scene if the DEBUG_VAR is set to true
That will prevent us forgetting this scene somewhere in the code